### PR TITLE
Create global imports file for Heath Care Application

### PR DESCRIPTION
**Pull request by longmd**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

This PR creates a global imports file to allow for things like the Schema JSON file, React bindings and other elements from the component library to be managed from a single source. This PR also normalizes the paths for platform imports.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#60897

## Acceptance criteria

 - Import paths are consistent across files
 - External imports are managed in a single file, to allow for easy swap, should their locations change

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution

### CODE REVIEW

File: src/applications/hca/config/chapters/veteranInformation/birthInformation.js [hunk 1]
```diff
@@ -32,8 +30,11 @@ export default {
           cityOfBirth,
           stateOfBirth: {
             type: 'string',
-            enum: [...states50AndDC.map(object => object.value), 'Other'],
-            enumNames: [...states50AndDC.map(object => object.label), 'Other'],
+            enum: [...STATES_50_AND_DC.map(object => object.value), 'Other'],
+            enumNames: [
+              ...STATES_50_AND_DC.map(object => object.label),
+              'Other',
+            ],
```
Comment (longmd) [inline src/applications/hca/config/chapters/veteranInformation/birthInformation.js:37]: These should really be moved to the schema file itself. No reason to be doing these here.

File: src/applications/hca/utils/imports.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import FULL_SCHEMA from 'vets-json-schema/dist/10-10EZ-schema.json';
+import SCHEMA_CONSTANTS from 'vets-json-schema/dist/constants.json';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import {
+  VaCheckbox,
+  VaModal,
+  VaSelect,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+// desctruct vets-json-schema constansts
```
Comment (coope93) [inline src/applications/hca/utils/imports.js:10]: small typos on this comment

File: src/applications/hca/utils/actions/enrollment-status.js [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
 import {
   HCA_ENROLLMENT_STATUSES,
   ENROLLMENT_STATUS_ACTIONS,
 } from '../constants';
-import {
-  callAPI,
-  callFake404,
-  callFakeSuccess,
-} from '../helpers/enrollment-status';
+import { callAPI, callFake404, callFakeSuccess } from '../helpers';
```
Comment (coope93) [inline src/applications/hca/utils/actions/enrollment-status.js:7]: I like this a lot 🙂

File: src/applications/hca/definitions/dependent.js [hunk 4]
```diff
@@ -17,7 +16,7 @@ const {
   dependents: {
     items: { properties: dependent },
   },
-} = fullSchemaHca.properties;
```
Comment (coope93) [inline src/applications/hca/definitions/dependent.js:20]: Did we drop the `hca` from this name since we are in the `hca` app folder already?

**Review by coope93 — DISMISSED** [review]

File: src/applications/hca/definitions/dependent.js (hunk 4 above)
Comment (longmd) [reply src/applications/hca/definitions/dependent.js:20]: Correct. It seems a bit redundant.

**Review by coope93 — APPROVED** [review]

**Pull request merged**